apps/governance/src/app.tsx

```diff
@@ -177,9 +183,10 @@ const AppContainer = () => {
   const { config, loading, error } = useEthereumConfig();
   const { VEGA_ENV, GIT_COMMIT_HASH, GIT_BRANCH, ETHEREUM_PROVIDER_URL } =
     useEnvironment();
+  const [telemetryOn] = useLocalStorage(TELEMETRY_ON);
 
   useEffect(() => {
-    if (ENV.dsn) {
+    if (ENV.dsn && telemetryOn) {
```

Comment:
chore (blocking): what about if the user switches during their session? We need an `else` block here to close the sentry connection with `Sentry.close()`